node/engine/chainservice/eth_chainservice.go

```diff
@@ -65,6 +74,15 @@ const MAX_QUERY_BLOCK_RANGE = 2000
 // This has been reduced to 15 seconds to support local devnets with much shorter timeouts.
 const RESUB_INTERVAL = 15 * time.Second
 
+// REQUIRED_BLOCK_CONFIRMATIONS is how many blocks must be mined before an emitted event is processed
+const REQUIRED_BLOCK_CONFIRMATIONS = 0
```

Comment:
This needs to be > 0 to work on Filecoin Calibration testnet. But it needs to set to 0 for our test suite to pass. We discussed a modification to the `AllocationUpdated` event that would allow the tests to pass when its set > 0.

Reply:
Did we explore the thought of having variable `REQUIRED_BLOCK_CONFIRMATIONS` that depend on the chain the chain service connects to?

Reply:
I think the better solution is to update the `AllocationUpdated` event so that we don't have to check chain state at a previous point in time (instead we can just parse the event for all the info we need). That would allow our tests to verify the block confirmation code is working as expected before we get to production. We discussed making the update to that event in the smart contract code, but are waiting to discuss the change with George/Colin next week

Reply:
Thinking about this a bit more. Updating the `AllocationUpdated` event and mapping different chain IDs to different values for `REQUIRED_BLOCK_CONFIRMATIONS` are related, but are addressing slightly different issues. We could go ahead and make both of those updates

Reply:
Created new issue to address variable `REQUIRED_BLOCK_CONFIRMATIONS` in a future PR: https://github.com/statechannels/go-nitro/issues/1535